src/repositories/AssetRepository.spec.ts

```diff
@@ -3,7 +3,8 @@
 import { afterEach, describe, expect, test, vi } from 'vitest';
 import { HttpClient } from '../utilities/httpClient';
 import { HttpUtilResponseDummy } from '../tests';
-import { AssetApiDummy, AssetDummy } from '../tests/AssetApiDummy';
+import { AssetApiDummy } from '../tests/AssetApiDummy';
+import { AssetDummy } from '../tests/AssetDummy';
```

Comment:
Couldn't we have here:
```suggestion
import { AssetApiDummy,  AssetDummy } from '../tests';
```